commons/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -140,7 +140,7 @@ protected Contribution doInBackground(Void... voids /* stare into you */) {
             @Override
             protected void onPostExecute(Contribution contribution) {
                 super.onPostExecute(contribution);
-                uploadService.queue(UploadService.ACTION_UPLOAD_FILE, contribution);
+                //uploadService.queue(UploadService.ACTION_UPLOAD_FILE, contribution);
```

Comment:
I don't think this should be committed, right? It is only a trick used while testing, I believe.